### PR TITLE
DRAFT - Offchain Incentives

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

**Review by hernandoagf — COMMENTED** [review]
Everything looks good on the technical side, just left a couple of minor non-tech related comments.

File: governance/proposal-bounties.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+# Off-chain Proposal Bounty Eligibility
+
+## Eligibility
+**1.1** - Any community member with an account on forum.makerdao.com is eligible to receive off-chain proposal bounty payouts.
+**1.2** - Community members must opt-in and provide an ethereum mainnet address in order to receive payouts. This address will be publicly visible.
+
+## Eligibility Exceptions
+**2.1** - Core Unit members are not eligible to recieve off-chain proposal bounties. This includes Core Unit Facilitators.
+**2.2** - Proposals concerning collateral onboarding are not eligible for off-chain proposal bounties.
```
Comment (TheExistence) [inline governance/proposal-bounties.md]: I'll add these two (2.2 and 2.3) to the forum post

File: governance/sourcecred.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+A pool of 2,500 DAI per week is allocated for distribution amongst active participants of the official MakerDAO forum.
+
+This pool comes out of the GovAlpha Core Unit's budget.
+
+### Cred Calculation
+
+SourceCred calculates reputation for users on the MakerDAO forum (called 'Cred') based on positive interactions between members of the community. Positive interactions, likes, responses, quotes, etc - result in the author recieving Cred.
+
+DAI is then allocated on a weekly basis according to two reward mechanisms.
```
Comment (TheExistence) [inline governance/sourcecred.md:17]: Maybe complement this sentence with a few words about the bias of the distribution formula? Something like "DAI is then allocated on a weekly basis according to two reward mechanisms with a higher weight for more recent contributions."?

File: governance/sourcecred.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+
+# Off-chain Participation Incentive Eligibility
+
+## Eligibility
+**1.1** - Any community member with an official forum account is eligible to receive SourceCred payouts.  
+**1.2** - Community members must opt-in and provide an ethereum mainnet address in order to receive payouts. This address will be publicly visible.  
+
+## Eligibility Exceptions
+**2.1** - Core Unit members are not eligible to recieve Community SourceCred payouts. This includes Core Unit Facilitators.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/sourcecred.md:47]: I'll add a point saying former members are eligible.

File: governance/delegate-sourcecred.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- Then, it filters out all of the actions that are not relevant for the scope of this calculation (user mentions, inter-week cred interactions, and cred lost for likes given).
+- Finally, it counts and stores the cred that is left in the selection.
+
+This makes the calculation more biased toward direct delegate activity and recent quality contributions, so Delegates that contribute less (garner fewer likes, replies, and link-backs) will notice that the DAI they receive from SourceCred will be greatly reduced.
+
+### Payout Process
+
+Engagement compensation in DAI is calculated on a weekly basis and paid out to Recognized Delegates once a month.
+
+Compensation calculations are transparent and located here: GovAlpha Observable.
```
[the remark this answers is not in the document]
Comment (LongForWisdom) [reply governance/delegate-sourcecred.md:35]: The visualization one!

**Pull request merged**